src/zenml/cli/model.py

```diff
@@ -630,15 +631,18 @@ def register_model_version(
             description: Description of the model to register.
             metadata: Model version metadata.
             zenml_version: ZenML version of the model to register.
-            zenml_pipeline_run_id: ZenML pipeline run ID of the model to register.
+            zenml_run_name: ZenML pipeline run name of the model to register.
+            zenml_pipeline_run_id: Deprecated. Use zenml_run_name instead.
             zenml_pipeline_name: ZenML pipeline name of the model to register.
             zenml_step_name: ZenML step name of the model to register.
         """
         # Parse metadata
         metadata = dict(metadata) if metadata else {}
         registerted_metadata = ModelRegistryModelMetadata(**dict(metadata))
         registerted_metadata.zenml_version = zenml_version
-        registerted_metadata.zenml_pipeline_run_id = zenml_pipeline_run_id
+        registerted_metadata.zenml_run_name = (
+            zenml_run_name or zenml_pipeline_run_id
+        )
         registerted_metadata.zenml_pipeline_name = zenml_pipeline_name
         registerted_metadata.zenml_step_name = zenml_step_name
```

Comment:
```suggestion
        registered_metadata.zenml_run_name = (
            zenml_run_name or zenml_pipeline_run_id
        )
        registered_metadata.zenml_pipeline_name = zenml_pipeline_name
        registered_metadata.zenml_step_name = zenml_step_name
```
Needs fixing elsewhere as well in the model, (i.e. above). I think.